pyproject.toml

```diff
@@ -42,13 +42,12 @@ dependencies = [
     "aiohttp",  # TODO: remove in favor of httpx
     "anyio",
     "coredis",
-    "fhaviary[llm]>=0.8.2",  # For namespace reorg
```

Comment:
Can you match this in `pre-commit`'s `mypy`

Reply:
good catch